RadixWallet/Features/AssetsFeature/AssetsView+Reducer.swift

```diff
@@ -157,6 +170,11 @@ public struct AssetsView: Sendable, FeatureReducer {
 					await send(.child(.poolUnitsList(.view(.refresh))))
 				}
 			}
+			if resourcesState.stakeUnitList != nil, state.activeAssetKind == .stakeUnits || state.isRefreshing {
```

Comment:
for non trivial boolean expressions like this I prefer a named variable defined above, or at least a comment :) `stakeUnitsIsRefereshing`?